redirects.yaml

```diff
@@ -13,6 +13,7 @@ ops-code-quality/?: /about
 overview/?: /about
 universal-operators/?: /about
 docs/(lma2|cos)(?P<page>.*)/?: https://charmhub.io/topics/canonical-observability-stack{page}
+/docs/juju/install-juju: /docs/juju/install-and-manage-the-client
```

Comment:
just noticed - do these need the `/` at the start?

Reply:
Oh, no, I suppose I don't